src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,95 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static StorageDaoImpl storage;
+    private static RegistrationServiceImpl service;
+
+    @BeforeAll
+    static void beforeAll() {
+        storage = new StorageDaoImpl();
+        storage.add(new User(1535854439L, "bobTheOne", "iAmHereAlice", 23));
+    }
+
+    @BeforeEach
+    void setUp() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_ExistingUser_NotOk() {
+        assertThrows(RuntimeException.class, () -> {
+            service.register(new User(1535854439L, "bobTheOne", "iAmHereAlice", 23));
+        });

```

Comment:
```suggestion
        assertThrows(RuntimeException.class, () -> 
            service.register(new User(1535854439L, "bobTheOne", "iAmHereAlice", 23))
        );

```
перевір в інших методах